tools/syz-trace2syz/parser/straceLex.rl

```diff
@@ -67,7 +67,7 @@ func (lex *Stracelexer) Lex(out *StraceSymType) int {
         *|;
 
         main := |*
-            [0-9]* => {out.val_int, _ = strconv.ParseInt(string(lex.data[lex.ts : lex.te]), 10, 64); tok = INT;fbreak;};
+            [1-9].[0-9]* => {out.val_int, _ = strconv.ParseInt(string(lex.data[lex.ts : lex.te]), 10, 64); tok = INT;fbreak;};
```

Comment:
Perhaps we could just use base `0` (auto-detection):
https://golang.org/pkg/strconv/#ParseInt
and remove the case for octal numbers.

Reply:
good suggestion